sonargraphintegration.properties

```diff
@@ -1,21 +1,21 @@
 #See also: Sonargraph.properties
 
-category=Additional Metrics
+category=External Analyzers
 description=Provides support for Sonargraph 8+'s architecture governance features, accompanied by metrics about cyclic dependencies and other structural aspects.
 homepageUrl=https://github.com/sonargraph/sonar-sonargraph-integration
 devVersion=
-publicVersions=2.1.7,4.0.0
-archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,3.1.0
+publicVersions=2.1.7,4.1.1
+archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,3.1.0,4.0.0
 privateVersions=
 
 defaults.mavenGroupId=com.hello2morrow
 defaults.mavenArtifactId=sonar-sonargraph-integration
 
-4.0.0.description=Single execution of plugin, compatibility with SonarQube 7.9.1
-4.0.0.sqVersions=[6.7.6,LATEST]
-4.0.0.date=2019-08-29
-4.0.0.changelogUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/tag/Release-4.0.0_with_releasenotes
-4.0.0.downloadUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-4.0.0_with_releasenotes/sonar-sonargraph-integration-4.0.0.jar
+4.1.1.description=Compatibility with SonarQube 8.1
+4.1.1.sqVersions=[7.9,LATEST]
```

Comment:
The comment says compatibility with 8.1. Shouldn't the compatibility range start there?

Reply:
The description means that the plugin runs now also with SonarQube 8.1, which did not work before. The plugin is still compatible with the lower versions of SonarQube.